src/main/java/com/lifeinide/jsonql/jpa/JpaFilterQueryBuilder.java

```diff
@@ -126,6 +126,18 @@ protected void init(CriteriaQuery<E> query, Root<?> root) {
 		return this;
 	}
 
+	@Nonnull
+	@Override
+	public JpaFilterQueryBuilder<E,P> add(@Nonnull String field, LikeQueryFilter filter) {
+		if (filter!=null) {
+			String pattern = filter.getPattern().toString();
+			Predicate predicate = pattern==null ? null : context.getCb().like(context.getRoot().get(field), pattern);
+			context.getPredicates().add(predicate);
+		}
```

Comment:
Currently `null` predicate can be added to `context.getPredicates()` and I don't know what happens then...

```suggestion
if (filter!=null && filter.getPattern()!=null)
  context.getPredicates().add(
    context.getCb().like(context.getRoot().get(field), filter.getPattern()));
```

Reply:
I've made the update to prevent the null predicate.  See 6494dca